bookkeeper-proto/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -112,6 +112,7 @@ message AddRequest {
     required int64 entryId = 2;
     required bytes masterKey = 3;
     required bytes body = 4;
+    optional int64 writeFlags = 5;
```

Comment:
are we planning to use 8 bytes? probably just int32?

Reply:
int32 should be good enough.

Reply:
done